packages/kokkos/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
   Kokkos_ENABLE_Cuda
   KOKKOS_HAVE_CUDA
   "Enable CUDA support in Kokkos."
-  "${KOKKOS_HAVE_CUDA_TPL}"
```

Comment:
I'm happy for the fix, but it worries me a bit that `KOKKOS_HAVE_CUDA_TPL` and `TPL_ENABLE_CUDA` don't appear to be connected.  Which one are we supposed to use?

Reply:
Apparently I switched it to `KOKKOS_HAVE_CUDA_TPL`... no idea what I was thinking, that variable isn't defined anywhere. There should be no future confusion because it doesn't exist anywhere else.